sunpy/net/dataretriever/tests/test_norh.py

```diff
@@ -16,26 +20,107 @@
 from sunpy.net.tests.strategies import time_attr, range_time
 
 
+NORHClient = norh.NoRHClient()
+
+
+def create_url(start, end, wavelength):
+
+    """
+    This function creates a url based on the NoRHClient data,
+    instead of making an online request.
+    """
+    freq = None
+    if wavelength == 34 * u.GHz:
+            freq = 'tcz'
+    elif wavelength == 17 * u.GHz:
+            freq = 'tca'
+    else:
+        raise ValueError('Wavelength should be 17Ghz or 34Ghz, found {}'.format(wavelength))
+    start = datetime.strptime(start, '%Y/%m/%d')
+    end = datetime.strptime(end, '%Y/%m/%d')
```

Comment:
Could you see if this is possible using `astropy.time.Time`?

Reply:
Is there any issue with datetime?

Reply:
No, we just moved to using `astropy.time.Time` and if possible we want to use that wherever possible. If it can not be done with `astropy.time.Time`, then `datetime` is fine.

Reply:
I think there are some issues with using `astropy.time.Time` as it does not support string representation.

Reply:
That's fine then.